client/package.json

```diff
@@ -815,7 +817,6 @@
     "unicode-match-property-value-ecmascript": "^2.1.0",
     "unicode-property-aliases-ecmascript": "^2.1.0",
     "unique-string": "^2.0.0",
-    "universal-cookie": "^7.1.0",
```

Comment:
Why did you remove this?

Reply:
I by mistake have done it but it was working from my end

Reply:
Ive added that! thanks